agent/eni/watcher/watcher_linux.go

```diff
@@ -337,7 +346,10 @@ func (udevWatcher *UdevWatcher) sendENIStateChangeWithRetries(parentCtx context.
 		sendErr := udevWatcher.sendENIStateChange(macAddress)
 		if sendErr != nil {
 			if _, ok := sendErr.(*unmanagedENIError); ok {
-				log.Debugf("Unable to send state change for unmanaged ENI: %v", sendErr)
+				// This can happen in two scenarios: (1) the ENI is indeed not managed by ECS (i.e. attached manually
+				// by customer); (2) this is an ENI attached by ECS but we have not yet received its information from
+				// ACS.
+				log.Debugf("Not sending state change because the ENI is not managed by ECS: %v", sendErr)
```

Comment:
nit: is "ENI is not managed by ECS" message valid if it's the second case? i think it might create customer confusion.

Reply:
i updated it to "Not sending state change because we don't know about the ENI". not sure if that sounds better?

Reply:
👍